.eslintrc

```diff
@@ -7,6 +7,8 @@
     "comma-dangle": ["error", "never"],
     "import/no-unresolved": "ignore",
     "no-param-reassign": ["error", {"props": false}],
-    "linebreak-style": 0
+    "linebreak-style": 0,
```

Comment:
Isn't it the change we made at the beginning to avoid problems with cross-platform development (macOS / Windows)? If I remember correctly it caused a problem with interpretation of every single new line in Windows...

Reply:
You are right. You just added `,` and I thought you introduced it in this commit. In general - we should use unix style, because inconsistency in this matter could lead to problem with code formatting.
To change it in VSC: https://stackoverflow.com/a/42643643/2051256

Reply:
Let's do this in a separate branch.